### DISCUSSION TITLE
browser's bookmarks OR list of urls

### DISCUSSION THREAD

**Discussion by tiger360** [espanso/espanso#1273] [Feature requests and Ideas]
Hey 😀, 

Is there a way yo browse a list of urls or bookmarks
and if I click on it, it will open the default browser.

![image](https://user-images.githubusercontent.com/5794444/169954830-fd5053c6-b005-437f-8254-11b7dff71fff.png)

Thanks

**Comment by federico-terzi** [comment]
Hey @tiger360,

Thank you for the feedback! Once macros #556 are implemented, this will become possible out of the box

Cheers :)
